### DISCUSSION TITLE
Using multiple domains for a single Epic Stack codebase

### DISCUSSION THREAD

**Discussion by timothymalcham** [epicweb-dev/epic-stack#704] [Q&A]
I'm building an app for a client, and I'm using the Epic Stack, deploying to fly.io. I'd like to keep everything together in one codebase, but the client would like to have some routes be assigned to one domain name, and other routes to a second domain name. I'd like to keep everything in a monolithic codebase since everything is still pretty small, and I need to share data between everything, so setting up APIs/RPC stuff seems like overkill.

I've never done anything like this before, so just trying to get a sense of where to start. I'm not trying to do multi-tenancy, but rather have a route (e.g. /route-a) map to a specific domain name (e.g. /route-a ---> example.com and /route-b ---> another.com) 

Would it be possible to handle this in the loaders for these routes? Possibly using `redirect(example.com/route-a)`? Would there be any major drawbacks or issues with this? Or is there a better approach?

**Comment by timothymalcham** [comment]
I see this thread on Fly.io about having two domains point to the same app: https://community.fly.io/t/how-to-use-two-domains-in-one-app/12015/9

I would likely try to configure this similarly, but hopefully have a way to manage what routes work with what domain via loaders.

**Reply by kentcdodds** [reply]
I'm sure you could do something like that. Your suggested approach sounds good to me as well.
